server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/event/ComputeMessagePreviewListenerTest.java

```diff
@@ -82,8 +87,8 @@ void setup() throws Exception {
 
         SessionProvider sessionProvider = new SessionProvider(authenticator, FakeAuthorizator.defaultReject());
 
-        ComputeMessagePreviewListener listener = new ComputeMessagePreviewListener(sessionProvider, messageIdManager,
-            messagePreviewStore, messageContentExtractor, htmlTextExtractor);
+        listener = spy(new ComputeMessagePreviewListener(sessionProvider, messageIdManager,
```

Comment:
I think we expect to be testing ComputeMessagePreviewListener behavior in the face of exception thrown by messageIdManager (for instance), the listener should not be your spy.